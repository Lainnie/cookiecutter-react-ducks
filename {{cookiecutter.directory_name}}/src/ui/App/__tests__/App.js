import React from 'react';
import shallow from 'enzyme/shallow';

import App from '../App';

describe('App', () => {
  it('renders correctly', () => {
    const tree = shallow(<App />)

    expect(tree).toMatchSnapshot();
  })
});
