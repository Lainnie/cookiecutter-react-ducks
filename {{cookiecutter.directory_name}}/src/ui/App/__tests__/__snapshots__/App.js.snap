// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h1
      className="App-title"
    >
      Welcome to React
    </h1>
  </header>
  <p
    className="App-intro"
  >
    From ui
  </p>
</div>
`;
